airbyte-integrations/connectors/source-google-ads/unit_tests/test_source.py

```diff
@@ -145,7 +145,7 @@ def test_chunk_date_range():
 def test_streams_count(config, mock_account_info):
```

Comment:
there should be unit tests for the newly added streams

Reply:
Done